src/browser/modules/Editor/EditorFrame.tsx

```diff
@@ -105,11 +111,15 @@ export function EditorFrame({
   )
   const editorRef = useRef<CodeEditor>(null)
 
-  function toggleFullscreen() {
-    setFullscreen(!isFullscreen)
-  }
+  const toggleFullscreen = useCallback(() => setFullscreen(!isFullscreen), [
+    isFullscreen
+  ])
 
-  useEffect(() => bus && bus.take(EXPAND, toggleFullscreen), [isFullscreen])
+  useEffect(() => bus && bus.take(EXPAND, toggleFullscreen), [
+    bus,
+    isFullscreen,
+    toggleFullscreen
```

Comment:
So this no longer need isFullscreen

Reply:
shall I remove it ?

Reply:
Yeah line 120 is not needed in this dependency array since it has the toggleFullscreen there instead.

Reply:
oh I see what you mean